### DISCUSSION TITLE
Mode7 and Fieldnumber Ideas

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#7083] [Ideas]
Would iit be possible to set an unload position like in the shovel mode, but for Bale Collector mode as well? Using a Fork that apparantly can be set by mode 7, the bale collector mode.

Of course, it would need a loading position as well

One more thing is would it be possible to make a new "field" using the override/create field number from recorded course instead of of a pre-existing field? It requires a field to calculate the perimeter to do so.

**Comment by Tensuko** [comment]
Shovel Mode unloads at Trailer and Trigger in front of the 3rd Waitpoint.
Bale Collecor unloads in Trigger (by drive through) or in reverse at Unloading Waypoints (see: https://www.youtube.com/watch?v=98TIUAgazZg )

Mode7 (bale collector) doesn't need a loading zone, it loads the bales that it finds on the field.
More otions won't come as bales behave strange enough in FS19.

You talk about recording a course as a Fieldborder ?
That might be an Idea, but is not planed yet.
I suggest you to plow your fieldborder and make a field out of it with the Numbers (btw, you can choose any number for the custom fields with scan field border.)

**Reply by rento112** [reply]
yeah, I just tested an autoload bale fork from the CSZ pack with the bale collector mode. It thinks the bale fork is a bale collector trailer, which is to be expected.
It would be totally awesome if you guys could make a new mode that combines shovel and bale collector mode so that we have the bale collector and the ability to unload it in a specific position like on the Bale Shredder for the Straw harvest add-on. needs a lift instead of dropping on the ground. That's' the idea I was asking about.

And the record ia course as a fieldborder would be quite nice. That's what I meant too
